UI/src/app/pages/mapping/store/reducers/comments.reducer.ts

```diff
@@ -0,0 +1,39 @@
+import { Action } from '@ngrx/store';
+import { ActionTypes, CommentsAction } from 'src/app/pages/mapping/store/actions/comments.actions';
+import { IComment } from '../../components/dialog/dialog.component';
+
+export interface Area {
+  [panelTitle: string]: Panel;
+}
+
+export interface Panel {
+  [rowName: string]: Row;
+}
+
+export interface Row {
+  comments: IComment[];
+}
+
+export interface State {
+    'source': Area;
```

Comment:
State is an object, simple objects in JS can have only string keys.
So when you use quotes - it's not a mistake. But try to use the same style of coding everywhere